envs/gubbins.yml

```
@@ -5,67 +5,5 @@ channels:
   - defaults
   - r
 dependencies:
-  - biopython=1.79=py39h89e85a6_1
-  - ca-certificates=2022.6.15=h033912b_0
-  - dendropy=4.5.2=pyh3252c3a_0
-  - dill=0.3.5.1=pyhd8ed1ab_0
-  - fasttree=2.1.10=hb4d813b_6
-  - freetype=2.10.4=h4cff582_1
-  - giflib=5.2.1=hbcb3906_2
-  - gmp=6.2.1=h2e338ed_0
-  - gubbins=3.2.1=py39pl5321h66c2135_0
-  - iqtree=2.0.3=h7eed37d_1
-  - jpeg=9e=h5eb16cf_1
-  - lcms2=2.12=h577c468_0
-  - lerc=3.0=he49afe7_0
-  - libblas=3.9.0=15_osx64_openblas
-  - libcblas=3.9.0=15_osx64_openblas
-  - libcxx=14.0.5=hce7ea42_1
-  - libdeflate=1.12=hac89ed1_0
-  - libffi=3.2.1=1
-  - libgfortran=5.0.0=9_3_0_h6c81a4c_23
-  - libgfortran5=9.3.0=h6c81a4c_23
-  - liblapack=3.9.0=15_osx64_openblas
-  - libllvm10=10.0.1=h009f743_3
-  - libopenblas=0.3.20=openmp_hb3cd9ec_0
-  - libpng=1.6.37=h7cec526_2
-  - libtiff=4.4.0=h9847915_1
-  - libwebp=1.2.2=h28dabe5_0
-  - libwebp-base=1.2.2=h0d85af4_1
-  - libxcb=1.13=h0d85af4_1004
-  - libzlib=1.2.12=hfe4f2af_1
-  - llvm-openmp=14.0.4=ha654fa7_0
-  - llvmlite=0.36.0=py39h798a4f4_0
-  - lz4-c=1.9.3=he49afe7_1
-  - mpi=1.0=openmpi
-  - multiprocess=0.70.13=py39h701faf5_1
-  - ncurses=6.3=h96cf925_1
-  - numba=0.53.1=py39h32e38f5_1
-  - numpy=1.22.4=py39h677350a_0
-  - openjpeg=2.4.0=h6e7aa92_1
-  - openmpi=4.0.5=h5807002_4
-  - openssl=1.1.1o=hfe4f2af_0
-  - perl=5.32.1=2_h0d85af4_perl5
-  - pillow=9.1.1=py39h579eac4_1
-  - pip=22.1.2=pyhd8ed1ab_0
-  - pthread-stubs=0.4=hc929b4f_1001
-  - python=3.9.0=ha017127_4_cpython
-  - python_abi=3.9=2_cp39
-  - rapidnj=2.3.2=hcd10b59_2
-  - raxml=8.2.12=ha5712d3_4
-  - raxml-ng=1.0.1=ha44586f_0
-  - readline=8.1.2=h3899abd_0
-  - reportlab=3.5.68=py39hf37cc50_1
-  - scipy=1.8.1=py39hfa1a3ab_0
-  - setuptools=62.3.4=py39h6e9494a_0
-  - ska=1.0=h6151dfb_3
-  - sqlite=3.38.5=hd9f0692_0
-  - tk=8.6.12=h5dbffcc_0
-  - tzdata=2022a=h191b570_0
-  - wheel=0.37.1=pyhd8ed1ab_0
-  - xorg-libxau=1.0.9=h35c211d_0
-  - xorg-libxdmcp=1.1.3=h35c211d_0
-  - xz=5.2.5=haf1e3a3_1
-  - zlib=1.2.12=hfe4f2af_1
-  - zstd=1.5.2=ha9df2e0_1
-prefix: /Users/wachsmannj/opt/anaconda3/envs/gubbins
+  - gubbins >= 3.1.0
+  - python =3.9.13
```

Remove python, it's 'special' in conda
```suggestion
  - gubbins >= 3.1.0
```